std_msgs/Header header
string[] joint_names
trajectory_msgs/JointTrajectoryPoint[] points
